src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -0,0 +1,292 @@
+import { nest } from 'd3-collection'
+import { graphql, useStaticQuery } from 'gatsby'
+import React, { useMemo, useState } from 'react'
+import cloneDeep from 'lodash/cloneDeep'
+
+import AreaChart from '../../components/charts/area-chart'
+
+import StatesWithPopulation from '../../data/visualization/state-populations.json'
+
+import {
+  getStateName,
+  parseDate,
+  totalColor,
+  positiveColor,
+} from '../../utilities/visualization'
+
+import './dashboard.scss'
+
+// these come from this google spreadsheet owned by Júlia Ledur
+// https://docs.google.com/spreadsheets/d/1mD_NhlJR1fM2Pv_pY8YixUrX2p2F8rAE0xPTtsTJOiM/edit#gid=0
+const stayAtHomeOrders = {
```

Comment:
This should really be in a separate file in `/src/data` or in our API eventually.